en/chronos.rst

```diff
@@ -115,7 +115,7 @@ Modifier Methods
 ----------------
 
 Chronos objects provide modifier methods that let you modify the value in
-a fine-grained way::
+a granular way::
```

Comment:
I understood better the previous way, but I'm gonna trust your native english skills @bcrowe :stuck_out_tongue: